package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "express-rate-limit",
-  "version": "5.3.0",
+  "version": "5.4.0",
```

Comment:
You should leave this out of the PR. After it's merged in, I'll use the `npm version minor` command which both increments the version number and makes a git tag.